spec/models/hyrax/collection_type_spec.rb

```diff
@@ -2,6 +2,14 @@
 RSpec.describe Hyrax::CollectionType, type: :model do
   let(:collection_type) { build(:collection_type) }
 
+  shared_context 'with a collection' do
+    let(:collection_type) { create(:collection_type) }
+
+    before do
+      FactoryBot.valkyrie_create(:hyrax_collection, collection_type_gid: collection_type.gid)
```

Comment:
Is there a reason for using `valkyrie_create` instead of `create(:collection_lw, collection_type_gid: collection_type.gid.to_s)` which is used elsewhere in the test (like line 155)?

Reply:
`:collection_lw` will go away when ActiveFedora does. it's a todo item to replace all the direct Fedora calls with valkyrie equivalents, so whenever we write new tests it's better to use it instead.